common/src/test/scala/uk/ac/wellcome/models/MergedSierraRecordTest.scala

```diff
@@ -7,236 +7,289 @@ import uk.ac.wellcome.utils.JsonUtil
-    val record = sierraItemRecord(
-      id = "i999",
-      title = "No, new narwhals are never naughty",
-      modifiedDate = "2009-09-09T09:09:09Z"
-    )
-    val mergedSierraRecord = MergedSierraRecord(
-      id = "b999",
-      itemData = Map(record.id -> record)
-    )
-
-    val newerRecord = sierraItemRecord(
-      id = record.id,
-      title = "Nobody noticed the naughty narwhals",
-      modifiedDate = "2010-10-10T10:10:10Z"
-    )
-    val result = mergedSierraRecord.mergeItemRecord(newerRecord).get
-
-    result.itemData.get(record.id).get shouldBe newerRecord
-  }
+      it("should update itemData when merging item records with newer data") {
+        val record = sierraItemRecord(
+          id = "i999",
+          title = "No, new narwhals are never naughty",
+          modifiedDate = "2009-09-09T09:09:09Z",
+          bibIds = List("b999")
+        )
+        val mergedSierraRecord = MergedSierraRecord(
+          id = "b999",
+          itemData = Map(record.id -> record)
+        )
 
-  it("should not overwrite existing itemData when there's a older update") {
-    val record = sierraItemRecord(
-      id = "i111",
-      title = "Only otters occupy the orange oval",
-      modifiedDate = "2001-01-01T01:01:01Z"
-    )
-    val mergedSierraRecord = MergedSierraRecord(
-      id = "b111",
-      itemData = Map(record.id -> record)
-    )
-
-    val newerRecord = sierraItemRecord(
-      id = record.id,
-      title = "Old otters outside the oblong",
-      modifiedDate = "2000-01-01T01:01:01Z"
-    )
-    val result = mergedSierraRecord.mergeItemRecord(newerRecord)
-    result shouldBe None
-  }
+        val newerRecord = sierraItemRecord(
+          id = record.id,
+          title = "Nobody noticed the naughty narwhals",
+          modifiedDate = "2010-10-10T10:10:10Z",
+          bibIds = List("b999")
+        )
+        val result = mergedSierraRecord.mergeItemRecord(newerRecord)
 
-  it("should support adding multiple items to a merged record") {
-    val record1 = sierraItemRecord(
-      id = "i111",
-      title = "Outside the orangutan opens an orange",
-      modifiedDate = "2001-01-01T01:01:01Z"
-    )
-    val record2 = sierraItemRecord(
-      id = "i222",
-      title = "Twice the turtles took the turn",
-      modifiedDate = "2002-02-02T02:02:02Z"
-    )
-
-    val mergedSierraRecord = MergedSierraRecord(id = "b121")
-    val result1 = mergedSierraRecord.mergeItemRecord(record1).get
-    val result2 = result1.mergeItemRecord(record2).get
-
-    result1.itemData.get(record1.id).get shouldBe record1
-    result2.itemData.get(record2.id).get shouldBe record2
-  }
+        result.itemData(record.id) shouldBe newerRecord
+      }
+
+      it("should return itself when merging item records with stale data") {
+        val record = sierraItemRecord(
+          id = "i111",
+          title = "Only otters occupy the orange oval",
+          modifiedDate = "2001-01-01T01:01:01Z",
+          bibIds = List("i111")
```

Comment:
bibId is wrong.

Reply:
interesting ... so this means that we can merge items to bibs without the correct bibids. I think this probably related to the unlinking problem, so i'm not going to add any more tests here.

Reply:
ah there are already tests for this `ignored` later on.